src/commands/game_commands/skip.ts

```diff
@@ -169,27 +169,28 @@ export default class SkipCommand implements BaseCommand {
         messageContext: MessageContext,
         interaction?: Eris.CommandInteraction
     ): Promise<void> {
-        if (interaction) {
-            await tryCreateInteractionSuccessAcknowledgement(
-                interaction,
-                LocalizationManager.localizer.translate(
-                    messageContext.guildID,
-                    "misc.interaction.genericProgress.title"
-                ),
-                LocalizationManager.localizer.translate(
-                    messageContext.guildID,
-                    "misc.interaction.genericProgress.description"
-                )
-            );
-        }
-
         const session = Session.getSession(messageContext.guildID);
 
         if (
             !session.round ||
             session.round.skipAchieved ||
             session.round.finished
         ) {
+            sendErrorMessage(
+                messageContext,
+                {
+                    title: LocalizationManager.localizer.translate(
+                        messageContext.guildID,
+                        "misc.failure.game.noneInProgress.title"
```

Comment:
This may confuse users if `session.round.skipAchieved || session.round.finished`